src/main/java/Duke.java

```diff
@@ -17,16 +17,6 @@ public Duke(String filePath) {
     }
 
 
-
-    public void Greet() {
```

Comment:
Should the method name start with small letters? So would it be better if it was `greet()` instead of `Greet()`

Reply:
Thanks for the comment. Will change the method name to start with small letters!